README.md

```diff
@@ -2,7 +2,7 @@
 [![Build Status](https://github.com/debricked/debricked-cli/actions/workflows/test.yml/badge.svg?branch=master)](https://github.com/debricked/debricked-cli/actions/workflows/test.yml)
 [![Latest Stable Version](https://poser.pugx.org/debricked/cli/v/stable)](https://packagist.org/packages/debricked/cli)
 
-![Debricked CLI in action](debricked-cli.png)
+![Debricked CLI in action](debricked-spaceship.svg)
```

Comment:
The spaceship is sexy, but I can kind of liked that we showed off the CLI (image could do with an update though and perhaps but somewhere else in the README)

Reply:
If kept like this, the ALT text needs to be changed

Reply:
I will create a updated one then

Reply:
ce3e9806def26df95389ab8dca0abaaa36b543ef @sweoggy

Reply:
deaffe87a3032a60de051371b3917f4ce7eae5cb @sweoggy